src/teleport.cpp

```diff
@@ -69,7 +75,7 @@ void Teleport::addThing(int32_t, Thing* thing)
 				return;
 			}
 
-			const Tile* tile = g_game.map.getTile(nextPos);
+			auto tile = g_game.map.getTile(nextPos);
```

Comment:
Could this be `const auto`
```suggestion
			const auto tile = g_game.map.getTile(nextPos);
```